modules/examples/docker-build/deployspec.yaml

```diff
@@ -1,11 +1,12 @@
+publishGenericEnvVariables: true
 deploy:
   phases:
     build:
       commands:
-        - aws ecr describe-repositories --repository-names ${AWS_CODESEEDER_NAME}-${ADDF_PARAMETER_ECR_REPO_NAME} || aws ecr create-repository --repository-name ${AWS_CODESEEDER_NAME}-${ADDF_PARAMETER_ECR_REPO_NAME} --image-scanning-configuration scanOnPush=true
+        - aws ecr describe-repositories --repository-names ${SEEDFARMER_PARAMETER_ECR_REPO_NAME} || aws ecr create-repository --repository-name ${SEEDFARMER_PARAMETER_ECR_REPO_NAME} --image-scanning-configuration scanOnPush=true
```

Comment:
NOTE: the only change from the original behaviour is ECR repository name used to be prefixed with seedkit name. I do not think this is required, but can change back to adhere to the same behaviour.